standards.md

```diff
@@ -55,7 +67,7 @@ Refer to the following set of questions to help fill the release-note section.
     * If this PR addresses a publicly known CVE, include the CVE number in the release notes
   * If unsure, include release note.
     * It's recommended to include release note explaining the changes in the PR.
-
+* If a feature is implemented incrementally, use a release note only when it is complete and ready to be used
```

Comment:
Thanks Lee! I love this new instruction, we should really only announce the a new feature in release note when it's done. (Nit: do we want to call out the API Change case? even though it's user facing, we shouldn;t mention it in the release note unless the implementation is in place.)

Reply:
I did actually :) take a look at the example under "incremental feature development"

Reply:
Ahh that looks great. Thanks Lee! Would it be helpful if we link that section ("Incremental Feature Development") here as a reference?

Reply:
done!